### ISSUE TITLE
wasm bindings cannot be built

### ISSUE THREAD

**Issue by turt2live** [matrix-org/matrix-rust-sdk#980]
```
> npm install

> @matrix-org/matrix-sdk-crypto-js@0.1.0-alpha.0 prepublish
> $npm_execpath run build && $npm_execpath run test

'$npm_execpath' is not recognized as an internal or external command,
operable program or batch file.
```

on Windows, nodejs 16.

**Comment by turt2live** [comment]
... and when you try to do it manually:

```
> npm run-script build

> @matrix-org/matrix-sdk-crypto-js@0.1.0-alpha.0 build
> cross-env RUSTFLAGS='-C opt-level=z' wasm-pack build --release --target nodejs --scope matrix-org --out-dir ./pkg

Error: Error during execution of `cargo metadata`: error: failed to run `rustc` to learn about target-specific information

Caused by:
  process didn't exit successfully: `rustc - --crate-name ___ --print=file-names '-C --target wasm32-unknown-unknown --crate-type bin --crate-type rlib --crate-type dylib --crate-type cdylib --crate-type staticlib --crate-type proc-macro --print=sysroot --print=cfg` (exit code: 1)
  --- stderr
  error: multiple input filenames provided (first two filenames are `-` and `'-C`)
```

**Comment by Hywan** [comment]
So apparently `$npm_execpath` doesn't exist on Windows, because why not.

And if you replace it by ˋnpm`, you get an error from ˋrustc`. Well. Which Rust version do you have please? I've never seen this error before.

**Comment by turt2live** [comment]
I'm running Rust 1.60.0, but can update if needed. 

`wasm-pack` when run under `npm` should be the one local to the project, though when I bypass `npm` it's the one from the user profile's bin directory.

**Comment by Hywan** [comment]
Please update to Rust 1.60, but I would be surprise if it solves the problem. I'll try to run the release script to see how things are going.

**Comment by Hywan** [comment]
About `$npm_execpath`, we should use `%npm_execpath%` on Windows. Because NPM scripts are super annoying to make interoperable, I will just force `npm` for the moment.

**Comment by Hywan** [comment]
Closed by #981 I guess. Please feel free to reopen if the problem isn't addressed.
